androidlibrary_lib/src/test/java/org/opendatakit/utilities/DateUtilsTest.java

```diff
@@ -46,4 +50,66 @@ public void testDateInterpretation() {
     assertEquals(expected, value.substring(0,expected.length()));
   }
 
+  @Test
+  public void testNowInput() {
+    TimeZone tz = TimeZone.getTimeZone(TimeZone.getAvailableIDs()[0]);
+    DateUtils util = new DateUtils(Locale.US, tz);
```

Comment:
You can move the `DateUtils` setup into a `@Before` method since it's used in all tests.